src/scanner/custom-rules/color-rule.ts

```diff
@@ -12,8 +12,8 @@ export const colorConfiguration: RuleConfiguration = {
         },
     ],
     rule: {
-        id: 'select-body',
-        selector: 'body',
+        id: 'select-document',
```

Comment:
While I think this solution works, I think a simpler solution may suffice. With regards to both uses of this rule, we only add the css styles to a single/first element in the `SingleTargetDrawer` (line 47 onwards `single-target-drawer.ts`).

In the webpage examples given, I believe the first element returned is actually the `html` element, so if we change the selector to `html` we should get the same result (and can also rename to `select-html`, per Dan's suggestion).